content/structure.md

```diff
@@ -0,0 +1,390 @@
+# Application structure and code style
+
+After reading this article, you'll know:
+
+1. How a Meteor application compares to other types of applications in terms of file structure.
+2. How to organize your application both for small and larger applications.
+3. How to format your code and name the parts of your application in consistent and maintainable ways.
+
+<h2 id="meteor-structure">Meteor Application Structure</h2>
+
+Meteor is a *full-stack* framework for building applications; this means Meteor applications differ from most applications in that they include code that both runs on the client, code that runs on the server, and code that runs in *both places*. Meteor applications enable you to run JavaScript code easily and consistenly in both client and server environments, and some conventions on application structure makes it easier to understand what code runs where.
+
+<h3 id="es2015-imports">ES2015 Imports</h3>
+
+As of version 1.3, Meteor ships with full support for ES2015 packages, both on the client and on the server. In particular it is simple and straightforward to include NPM package code in either environment, and to write your code in a modular way, avoiding the introduction of global symbols and "action at a distance". 
```

Comment:
Also, I think we should greatly expand this section to be an introduction to modules in general, for people who aren't aware that a module is really a file, that each file can export one or more items, that some modules can be shared between client and server, etc.

We don't have anywhere else that explains this stuff, so this is the place.